lib/wasmboyControllerState.js

```diff
@@ -1,11 +1,51 @@
+const Key = {
```

Comment:
This is a way better way of doing things, thanks!

I totally wrote up with gamepad thing in like 30 minutes haha, glad to see it looking better :)